src/Init/Data/Nat/Bitwise/Basic.lean

```diff
@@ -78,6 +78,8 @@ of a number.
 -/
 
 /-- `testBit m n` returns whether the `(n+1)` least significant bit is `1` or `0`-/
-def testBit (m n : Nat) : Bool := (m >>> n) &&& 1 != 0
+def testBit (m n : Nat) : Bool :=
+  -- `1 &&& n` is faster than `n &&& 1` for big `n`. This may change in the future.
```

Comment:
I'm not sure we need this comment in the code, but it would be good to add it to the PR description as explanation for why this change was made.
Presumably the only thing that might change in the future is that `n &&& 1` will be similarly optimized, not that suddenly `n &&& 1` will be significantly faster than `1 &&& n`, so the "this may change in the future" part seems unneeded?